_rules/SC2-4-6-descriptive-headings.md

```diff
@@ -10,6 +10,7 @@ success_criterion:
 
 test_aspects:
 - DOM Tree
+- Accessibility Tree
```

Comment:
Add "CSS Styling", since e.g. "visible" is included in applicability.

Not sure if "accessibility tree" needs to be listed, or if that can be inferred from the other two. We haven't listed it in other rules.

Reply:
Added CSS Styling. We must keep Accessibility Tree,  since the rule applies to visual hidden text that is only in the Accessibility Tree.

Reply:
I know, but they are still in the DOM, which is also listed. See e.g. the published "Iframe has accessible name" rule, https://auto-wcag.github.io/auto-wcag/rules/SC4-1-2-iframe-has-name.html.
If we need to include "Accessibility Tree" for this rule, a lot of other rules have to be updated too. I don't think any of our other rules has "Accessibility Tree" listed under test aspects.

Reply:
I guess we can do that : changing to DOM Tree and CSS Styling then.